pages/emotions/[slug]/index.js

```diff
@@ -22,6 +22,15 @@ export default function EmotionDetailsPage() {
   }
   const { slug } = router.query;
   const emotion = emotionData.find((emotion) => emotion.slug === slug);
+  const emotionIndex = emotionData.findIndex(
+    (emotion) => emotion.slug === slug
+  );
+
+  let prevEmotion = emotionData[emotionIndex - 1];
+  emotionIndex === 0 ? (prevEmotion = emotionData[6]) : prevEmotion;
+  let nextEmotion = emotionData[emotionIndex + 1];
+  emotionIndex === 6 ? (nextEmotion = emotionData[0]) : nextEmotion;
```

Comment:
When using conditional assignment, directly initializing `prevEmotion` and `nextEmotion` would provide clearer and more easily understandable logic. It's advisable to avoid overwriting variables unnecessarily for improved readability.

```suggestion
  const prevEmotionIndex = emotionIndex === 0 ? emotionData.length - 1 : emotionIndex - 1;
  const prevEmotion = emotionData[prevEmotionIndex];

  const nextEmotionIndex = emotionIndex === emotionData.length - 1 ? 0 : emotionIndex + 1;
  const nextEmotion = emotionData[nextEmotionIndex];
```